pkg/build/util/util.go

```diff
@@ -0,0 +1,12 @@
+package util
+
+import (
+	"fmt"
+
+	buildapi "github.com/openshift/origin/pkg/build/api"
+)
+
+// GetBuildPodName returns name of the build pod.
+func GetBuildPodName(build *buildapi.Build) string {
+	return fmt.Sprintf("pod-%s", build.Name)
```

Comment:
@mfojtik  @smarterclayton has been telling us to never use "1"+"2" and always use fmt.Sprintf.  i'm with you, though.